services/cloudrun-service/pkg/cloudrun/cloudrun_test.go

```diff
@@ -0,0 +1,305 @@
+/*This file is part of kuberpult.
+
+Kuberpult is free software: you can redistribute it and/or modify
+it under the terms of the Expat(MIT) License as published by
+the Free Software Foundation.
+
+Kuberpult is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+MIT License for more details.
+
+You should have received a copy of the MIT License
+along with kuberpult. If not, see <https://directory.fsf.org/wiki/License:Expat>.
+
+Copyright 2023 freiheit.com*/
+
+package cloudrun
+
+import (
+	"testing"
+
+	"google.golang.org/api/run/v1"
+)
+
+const errorMessage = "Expected %s, but got %s"
+
+func TestIsPresent(t *testing.T) {
+	for _, test := range []struct {
+		testName         string
+		servicesResponse []*run.Service
+		serviceName      string
+		expectedResponse bool
+	}{
+		{
+			testName:         "No services returned",
+			servicesResponse: []*run.Service{},
+			serviceName:      "myService",
+			expectedResponse: false,
+		},
+		{
+			testName: "Service does not exist",
+			servicesResponse: []*run.Service{
+				{Metadata: &run.ObjectMeta{Name: "notMyService"}},
+				{Metadata: &run.ObjectMeta{Name: "alsoNotMyService"}},
+				{Metadata: &run.ObjectMeta{Name: "notIt"}},
+			},
+			serviceName:      "myService",
+			expectedResponse: false,
+		},
+		{
+			testName: "Service exists",
+			servicesResponse: []*run.Service{
+				{Metadata: &run.ObjectMeta{Name: "notMyService"}},
+				{Metadata: &run.ObjectMeta{Name: "myService"}},
+				{Metadata: &run.ObjectMeta{Name: "notIt"}},
+			},
+			serviceName:      "myService",
+			expectedResponse: true,
+		},
+	} {
+		testCase := test
+		t.Run(testCase.testName, func(t *testing.T) {
+			t.Parallel()
+			exists := isPresent(testCase.servicesResponse, testCase.serviceName)
+			if exists != testCase.expectedResponse {
+				t.Errorf("Expected %t, but got %t", testCase.expectedResponse, exists)
+			}
+		})
+	}
+}
+
+func TestGetParent(t *testing.T) {
+	for _, test := range []struct {
+		testName       string
+		service        *run.Service
+		expectedParent string
+		expectedError  string
```

Comment:
please use `error` here 
```suggestion
		expectedError  error
```
See https://github.com/freiheit-com/kuberpult/blob/main/README_DEVELOPERS.md#always-use-cmpdiff
for how to compare errors.